### PR TITLE
Updates DG user profile, value prop and stories

### LINKED ISSUE #7: Update DG with user stories

### LINKED ISSUE #11: Fix broken header formatting in UG
Need to leave a space after # to specify header text.

### PR DESCRIPTION
**Pull request by euph00**
This will fix the header formatting bug in the UG (fixes #11) and updates the user profile, value proposition and user stories in the DG (fixes #7).

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -24,180 +24,180 @@ reliably than traditional GUI based apps.
...
 Lists all existing wards on the GUI. Only ward names will be displayed.
 
-###Format: `lsward`
+### Format: `lsward`
 
 ---
-##Update patient particulars: editpat
+## Update patient particulars: editpat
 This overwrites the specified patient’s particulars with new particulars. The patient to be edited must exist, i.e. there must exist a
 patient with the given `PATIENT_ID_NUMBER`. **All existing information about the patient will be deleted, and replaced with
```
Comment (pzhengze) [inline docs/UserGuide.md:142]: Shouldn't we only overwrite particulars that are specified?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -24,180 +24,180 @@ reliably than traditional GUI based apps.
...
 the new information.**
```
Comment (pzhengze) [inline docs/UserGuide.md:143]: Shouldn't we only overwrite particulars that are specified?

Comment (euph00) [reply docs/UserGuide.md:143]: This is for 1.2 where we implement minimum feature set. Will probably have to change the implementation for 1.3 anyway, no need for 1.2 to get too complicated. 

Although if not too troublesome to implement overwrite by necessity the UG can of course be updated to reflect as such. This is up to whoever is implementing this, I came up with this set of functionality as guide only without too much consideration

Comment (pzhengze) [reply docs/UserGuide.md:143]: Ok no issue then.

**Review by pzhengze — APPROVED** [review]
Looks good.

**Pull request merged**